src/components/form/adapters/widget-adapter.ts

```diff
@@ -109,20 +110,23 @@ export class LimeElementsWidgetAdapter extends React.Component {
             ...events,
         };
 
-        return React.createElement(LimeElementsAdapter, {
-            name: name,
-            elementProps: {
-                value: value,
-                label: this.getLabel(),
-                disabled: disabled,
-                readonly: readonly,
-                required: this.isRequired(),
-                invalid: this.isInvalid(),
-                'helper-text': this.getHelperText(),
-                ...extraProps,
-            },
-            events: newEvents,
-        });
+        return [
+            React.createElement(LimeElementsAdapter, {
+                name: name,
+                elementProps: {
+                    value: value,
+                    label: this.getLabel(),
+                    disabled: disabled,
+                    readonly: readonly,
+                    required: this.isRequired(),
+                    invalid: this.isInvalid(),
+                    'helper-text': this.getHelperText(),
+                    ...extraProps,
+                },
+                events: newEvents,
+            }),
+            getHelpComponent(this.props.widgetProps.schema),
```

Comment:
@vicgeralds could you take a look at the mechanics surrounding how we insert the help component in the DOM?

Reply:
0aa08cf68e1905e1dba4b720ee48236b1a496369